eve/generate_junit_result.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Simple ugly bash script to generate really really simple junit file
+# with just the result of the build step, waiting for something integrated
+# directly in `eve`
```

Comment:
Don't when something will land in Eve, but in the meantime, can we use "real" tooling instead of some half-assed XML rendering from Bash? Something like [python-junit-xml](https://github.com/kyrus/python-junit-xml) could be nice :)

Reply:
Then we need to install a new package (only available from epel btw) for a really simple xml for me it's not worth for this first step

Reply:
Honestly, I'd rather have a single step running after a full Eve build in a `kube_pod` worker, with the dependencies it needs to generate whatever report TestRail needs. But yeah, no need for this PR.